app/gui/src/controller/searcher.rs

```diff
@@ -1218,7 +1218,8 @@ impl Searcher {
             let requests = return_types_for_engine.into_iter().map(|return_type| {
                 info!("Requesting suggestions for returnType {return_type:?}.");
                 let file = graph.module.path().file_path();
-                ls.completion(file, &position, &this_type, &return_type, &tags)
+                let is_static = if this_type.is_some() { Some(false) } else { None };
```

Comment:
Please put it outside the closure (as I suggested) - there is no need to recompute it for every return_type.

Also, the `if` may be replaced with `then_some` method: https://doc.rust-lang.org/std/primitive.bool.html#method.then_some